internal/fs/fs.go

```diff
@@ -25,6 +25,9 @@ const (
 
 	// DefaultSysMountPoint is the common mount point of the sys filesystem.
 	DefaultSysMountPoint = "/sys"
+
+	// DefaultConfigfsMountPath is the commont mount point of the configfs
```

Comment:
The comment should be updated to match the variable name.

Reply:
Get it, done, pushed. Thanks